src/components/confirm/confirm.spec.js

```diff
@@ -306,5 +293,174 @@ describe("Confirm", () => {
         { modifier: `${StyledIcon}` }
       );
     });
+
+    it("confrim and cancel buttons should be `destructive`", () => {
```

Comment:
Spelling mistake here. Should be `confirm`.

Reply:
good point, thank you, changed